packages/@glimmer/debug/lib/metadata.ts

```diff
@@ -54,11 +54,8 @@ export type RawOperandFormat = OperandName | PresentArray<OperandName>;
 export function normalize(key: string, input: RawOperandMetadata): NormalizedMetadata {
   let name: ShorthandOperand;
 
-  if (input.format === undefined) {
-    throw new Error(`Missing format in ${JSON.stringify(input)}`);
-  }
-
   if (Array.isArray(input.format)) {
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion -- @fixme
```

Comment:
I left quite a few of these `!` assertions alone with the intent of going back and triaging them. I
still want to do a quick pass for cases where an alternative pattern would be better, but I
generally tried to break ties by leaving the behavior alone. I think it would be good to follow up
and try to be more precise about what's going on in these situations, but I think it's ok to do so
incrementally.